lib/helpers.ex

```diff
@@ -13,6 +13,20 @@ defmodule Helpers do
     end)
   end
 
+  @doc """
+  Apply a function to all of the keys in a map
```

Comment:
@skyqrose with a function like this, it's good to say what happens if you end up mapping two different keys to the same result. Which if I'm reading this right, it would pick an arbitrary one.